tests/core/method-class/test_method.py

```diff
@@ -94,16 +94,15 @@ def test_input_munger_falsy_config_result_in_default_munger():
         mungers=[],
         json_rpc_method='eth_method',
     )
-    method.input_munger(object(), [], {}) == []
+    assert method.input_munger(object(), [], {}) == []
 
 
-def test_default_input_munger_with_input_parameters_exception():
+def test_default_input_munger_with_input_parameters():
     method = Method(
         mungers=[],
         json_rpc_method='eth_method',
     )
-    with pytest.raises(TypeError):
-        method.input_munger(object(), [1], {})
+    assert method.input_munger(object(), [1], {}) == [1]
```

Comment:
Well, `mungers=None` is the same as `mungers=[]` in this case since with no `mungers` it just passes the raw value of the param through and returns the result. I think we should probably parametrize all empty cases here though for clarity.

Thanks for raising this. I actually see quite a few improvements to the testing that can be made. I'll add a commit with these in there.

What we can test separately